middleman-core/lib/middleman-core/step_definitions/commandline_steps.rb

```diff
@@ -0,0 +1,15 @@
+When /^I stop middleman if the output contains:$/ do |expected|
```

Comment:
This contains some code of an upcomming `aruba`-version. I will let you know when/how it can be safely removed. I still working on that bits over at cucumber/aruba.